NetCasbin/Persist/Helper.cs

```diff
@@ -5,7 +5,7 @@ namespace NetCasbin.Persist
 {
     public class Helper
     {
-        public delegate void LoadPolicyLineHandler<T, U>(T t, U u);
+        public delegate void LoadPolicyLineHandler<T, TU>(T t, TU u);
```

Comment:
Any reason to use `TU` here? I think `U` is a good name for a generic type parameter, like typescript or rust does.

Reply:
It is suggested using start with T at the generic parameter name and the name should express clear meaning as much as possible when the number of parameters is not one.
So I think here is the best:
```csharp
public delegate void LoadPolicyLineHandler<in TLine, in TModel>(TLine line, TModel model);
```
Actually, I am not very clear about the reason for using the `delegate` here now, It is only used by private API now.

Reply:
Sounds reasonable and use delegate here is fine because LoadPolicyLineHandler is passed as a parameter to other methods.

Reply:
Why do not use `Action<T1, T2>`here?

Reply:
I think it's ok to use `Action` at `LoadPolicyData`, the same effect.

Reply:
I will change it at another pull request.